Assignment6/Q1.py

```diff
@@ -0,0 +1,103 @@
+# coding=utf-8
+"""The solution presented here goes through all parking spaces and 'places' on each of them the desired car (car which
+   on the particular space in the desired positions layout - second array). Because dictionary structure is used to
+   keep track of current cars' positions the runtime of the given solution is θ(N) where N is the number of car and the
+   space used is also θ(N)."""
+
+from Move import Move
+
+
+def create_position_map(current_positions):
+    """The method creates a dictionary which assigns each car its current position.
+
+        Args:
+            current_positions: an array of integers, indicates the current locations (layout) of cars on the parking
+
+        Returns:
+            A dictionary mapping integers to integers. Cars' numbers are keys and cars' position on the parking
+            are values.
+    """
+    position_map = {}
```

Comment:
You don't really need to use `dict` here. You can use `list` instead of it, because our keys are `0`..`n-1`.
I don't sure, but it seems for me, that `dict` should take more space than `list`, because it uses hash-table.